src/AnnotationDialog.js

```diff
@@ -373,8 +370,7 @@ class AnnotationDialog extends EventEmitter {
 
         if (!this.isMobile) {
             this.element.addEventListener('click', this.clickHandler);
-            this.element.addEventListener('mouseenter', this.mouseenterHandler);
-            this.element.addEventListener('mouseleave', this.mouseleaveHandler);
+            this.element.addEventListener('click', this.stopPropagation);
```

Comment:
Why is this needed? And why not just add stopPropagation() to the clickHandler?

Reply:
Turns out it's already there in clickHandler